custom_components/tuya_local/devices/littepets.yaml

```diff
@@ -0,0 +1,121 @@
+name: Cat litter box
+products:
+  - id: 000004cha8
```

Comment:
Is this id the product id? Or is it a model number?

Reply:
From the logs:
"model": "{\"modelId\":\"000004cha8\"
I assumed it was the product ID. Any idea where to find the product ID?

Reply:
Product id is usually listed in the same tuya cloud api calls that return the local key

Reply:
"product_id": "jdxbiz2ca8x2mbfk"
Now added to the yaml